security/authn_authz.adoc

```diff
@@ -0,0 +1,67 @@
+[[security-auth]]
+= Authentication and Authorization
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:sourcedir: ../
+
+toc::[]
+
+[[security-auth-controlling-access]]
+== Controlling Access Using OAuth
+
+API access control via authentication and authorization is critical for securing
+your container platform. The {product-title} master includes a built-in OAuth
+server. Developers and administrators obtain OAuth access tokens to authenticate
+themselves to the API.
+
+As an administrator, you can configure OAuth to authenticate using the _identity
+provider_ of your choice, including:
+
+- Keystone
+- LDAP
+- Basic Authentication
+- GitHub / GitLab
+- Google
+- OpenID Connect
+- and more.
+
+[[security-auth-secure-self-service-web-console]]
+== Secure Self-service Web Console
+
+One of the key values of a container platform is the ability to enable developer
+self-service, making it easier and faster for your development teams to deliver
+applications built on approved layers. Multi-tenancy security is a must for the
+platform itself to make sure teams do not access each other's environments
+without authorization.
+
+{product-title} provides a self-service web console that gives enough control to
+teams to foster collaboration while still providing security. {product-title}
+adds several components to Kubernetes to ensure a secure multi-tenant master:
+
+- All access to the master is over TLS
+- Access to the API Server is X.509 certificate or token based
+- Project quota is used to limit how much damage a rogue token could do
+- Etcd is not exposed directly to the cluster
+
+[[security-auth-further-reading]]
+== Further Reading
+
+- _{product-title} Installation and Configuration_:
+xref:{sourcedir}install_config/configuring_authentication.adoc#install-config-configuring-authentication[Configuring Authentication and User Agent]
+** Includes full list of identity providers
+- _{product-title} Architecture_
+** xref:{sourcedir}architecture/additional_concepts/authentication.adoc#architecture-additional-concepts-authentication[Additional
+Concepts -> Authentication]
+** xref:{sourcedir}architecture/additional_concepts/authorization.adoc#architecture-additional-concepts-authorization[Additional
+Concepts -> Authorization]
+** xref:{sourcedir}architecture/infrastructure_components/web_console.adoc#architecture-infrastructure-components-web-console[Infrastructure Components -> Web Console]
+- _{product-title} Developer Guide_
+** xref:{sourcedir}dev_guide/authentication.adoc#web-console-authentication[Web Console Authentication]
+** xref:{sourcedir}dev_guide/authentication.adoc#cli-authentication[CLI Authentication]
+- xref:{sourcedir}cli_reference/index.adoc#cli-reference-index[_{product-title} CLI Reference_]
```

Comment:
@bfallonf Nah, I was just testing out the idea in this PR, since it's self-contained within this new guide. Been planning to bring it up as a discussion point in a future meeting after I tried it out more.